language/vm/src/file_format.rs

```diff
@@ -1144,6 +1144,10 @@ impl Bytecode {
 
     /// Return the successor offsets of this bytecode instruction.
     pub fn get_successors(pc: CodeOffset, code: &[Bytecode]) -> Vec<CodeOffset> {
+        checked_precondition!(
+            pc <= u16::max_value() - 2 && (pc as usize) < code.len(),
```

Comment:
where would I look to make myself smarter about the choice of bounds ? could a comment give me context?

Reply:
I will add a comment to make this more clear, thanks.